frontend/vue.config.js

```diff
@@ -15,10 +17,26 @@ const productionPlugins = [
       // so we create a snapshot of the page after the
       // `data-view` attribute exists in the DOM.
       renderAfterElementExists: '[data-view]'
-    })
+    }),
+    postProcess (renderedRoute) {
+      // Ignore any redirects.
+      renderedRoute.route = renderedRoute.originalRoute
+      // Basic whitespace removal. (Don't use this in production.)
+      renderedRoute.html = renderedRoute.html.split(/<div id="app"/)
```

Comment:
this has been added to index.html already

Reply:
Oh this is a quote I copy-pasta'd. Just wanted to see if this works to stop re-render.